Add a few plots with collapsed ages and mean ages over time

bxq98 opened the pull request:
Hi, @nmarquezuw! I added a few more plots according to your previous comments. I will keep thinking of ways to visualize the data and if I make any progress on that I'll make another pull request by the end of today.

Commit:
Merge pull request #1 from nmarquezuw/master

Merge pull request #2 from bxq98/master

stevenBXQ committed:
Add a few plots with collapsed ages and mean ages over time

stevenBXQ committed:
edit comments

stevenBXQ committed:
Add "download_kc_median_household_income" function

ghost commented:
The latest commit today (Add "download_kc_median_household_income" function bd85df3) include a function named "download_kc_median_household_income". It could download the median household income using the ACS API

nmarquezuw committed:
Update 00_utils.R 

Made some minor changes to download median household income data from the Census API so that it can download data from any state county combination.
